packages/upstream/cohttp-async.2.3.0/opam

```diff
@@ -24,21 +24,23 @@ doc: "https://mirage.github.io/ocaml-cohttp/"
 bug-reports: "https://github.com/mirage/ocaml-cohttp/issues"
 depends: [
   "ocaml" {>= "4.04.1"}
-  "dune" {build & >= "1.1.0"}
+  "dune" {>= "1.1.0"}
   "async_kernel" {>= "v0.11.0"}
   "async_unix" {>= "v0.11.0"}
   "async_extra" {>= "v0.11.0"}
   "base" {>= "v0.11.0"}
   "core" {with-test}
-  "cohttp"
+  "cohttp" {=version}
```

Comment:
Is this correct? I assume this enforces that co-located package are all on the same version.

Reply:
Yes, we are starting using this in some packages that have very strong interdependency upstream